src/web_socket_server_internal.cpp

```diff
@@ -192,6 +192,38 @@ bool WebSocketServerInternal::sendMessage(uint32_t conn_id, const void* payload,
   return result;
 }
 
+bool WebSocketServerInternal::sendMessageAll(const char* payload) {
+  if (connection_by_id.size() == 0) {
+    DEBUG("connection map is empty");
+    return false;
+  }
+
+  cyw43_thread_enter();
+  
+  for (auto& kv: connection_by_id) {
+    kv.second->sendWebSocketMessage(payload);
```

Comment:
```suggestion
  for (const auto& [_, connection] : connection_by_id) {
    connection->sendWebSocketMessage(payload);
```

Reply:
I have used structured bindings for the loops as suggested.